tests/accumulator_test.py

```diff
@@ -22,13 +24,13 @@ def test_with_mean_and_sum_squares(self):
         self.assertEqual(computed_metrics, [3.5, 25])
 
     def test_adding_accumulator(self):
-        mean_acc1 = MeanAccumulator().add_value(5)
-        sum_squares_acc1 = SumOfSquaresAccumulator().add_value(5)
+        mean_acc1 = MeanAccumulator(params=None, values=[5])
```

Comment:
I like having `params` and `values` as keyword arguments here, it increases readability significantly

Reply:
Thanks! agreed!